src/integrations/terminal/standalone/StandaloneTerminalProcess.ts

```diff
@@ -305,40 +319,56 @@ export class StandaloneTerminalProcess extends EventEmitter<TerminalProcessEvent
 	}
 
 	/**
-	 * Terminate the process if it's still running.
+	 * Terminate the process and all its children.
+	 *
+	 * On Unix: Uses process groups to kill the entire process tree.
+	 * When spawned with detached: true, the process becomes the leader of its own
+	 * process group. Using process.kill(-pid) sends the signal to all processes
+	 * in that group, ensuring child processes (like webpack-dev-server) are killed.
+	 *
+	 * On Windows: Uses taskkill with /T flag to kill the process tree.
 	 */
 	terminate(): void {
 		if (!this.childProcess || this.isCompleted) {
-			console.log(`[StandaloneTerminal] Process already completed or doesn't exist, skipping termination`)
 			return
 		}
 
 		const pid = this.childProcess.pid
-		console.log(`[StandaloneTerminal] Terminating process ${pid} with SIGTERM`)
+		if (!pid) {
+			return
```

Comment:
Should we try to kill it still?

```suggestion
			this.childProcess.kill("SIGTERM")
			return
```